app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -115,7 +115,9 @@ export default {
         [`desc`, `asc`]
```

Comment:
Not sure how long this list can become (if less then 1000 ignore the following please :) )

Ordering is more expensive than filtering, so maybe we can order on top of the filtered list

Reply:
On PageWallet we don't need to care about this for now. But very good insight!